travis-ci/vmtest/build_pahole.sh

```diff
@@ -9,6 +9,7 @@ travis_fold start build_pahole "Building pahole"
 CWD=$(pwd)
 REPO_PATH=$1
 PAHOLE_ORIGIN=https://git.kernel.org/pub/scm/devel/pahole/pahole.git
+PAHOLE_BRANCH=${PAHOLE_BRANCH:-master}
```

Comment:
you probably meant to use $(PAHOLE_BRANCH} below in `git checkout master`, right?